pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1273,12 +1273,13 @@ protected void handleProducer(final CommandProducer cmdProducer) {
                     return null;
                 }
 
+                // Do not print stack traces for expected exceptions
                 if (cause instanceof NoSuchElementException) {
                     cause = new TopicNotFoundException("Topic Not Found.");
-                }
-                if (!Exceptions.areExceptionsPresentInChain(cause,
+                    log.error("[{}] Failed to load topic {}, producerId={}: Topic not found", remoteAddress, topicName,
```

Comment:
Since it is an exception that can be ignored, can we use info?

Reply:
+1

Reply:
Yes, I completely agree. Updating now.